src/Client.php

```diff
@@ -157,12 +163,12 @@ public function user(bool $detailed = false): UserResponse
     /**
      * {@inheritdoc}
      */
-    public function userBalance(string $report_type_uid, bool $detailed = false): UserBalanceResponse
+    public function userBalance(UserBalanceParams $params): UserBalanceResponse
     {
         return UserBalanceResponse::fromHttpResponse(
-            $this->doRequest(new Request('get', \sprintf('user/balance/%s', \urlencode($report_type_uid))), [
+            $this->doRequest(new Request('get', \sprintf('user/balance/%s', \urlencode($params->getReportTypeUid()))), [
                 'query' => [
-                    '_detailed' => $detailed
+                    '_detailed' => $params->isDetailed()
```

Comment:
```suggestion
                    '_detailed' => $params->isDetailed() === true
```

Reply:
Yep, I fixed.

At this point, we have a some different logic:
```php
    'query' => [
        '_content'  => $params->isIncludeContent() === false
            ? 'false'
            : 'true',
        '_detailed' => $params->isDetailed() === false
            ? 'false'
            : 'true',
    ]
```

Reply:
It seems like not fixed now